crates/sp-domains-fraud-proof/src/storage_proof.rs

```diff
@@ -437,12 +457,25 @@ impl DomainInherentExtrinsicDataProof {
             domain_id,
             storage_key_provider,
         )?;
+
+        let maybe_domain_sudo_call_proof = if should_include_domain_sudo_call {
```

Comment:
Would be better to add a comment here to explain `should_include_domain_sudo_call` is only related to the consensus runtime version.